### DISCUSSION TITLE
EA0326GMP刷新sysupgrade失败

### DISCUSSION THREAD

**Discussion by Tian-Jionglu** [immortalwrt/immortalwrt#1228] [Q&A] [answered]
Hi，
参考 https://github.com/openwrt/openwrt/pull/12820 的方法刷新EA0326GMP固件：
1. 开启SSH；
2. 从[ImmortalWrt-Selector](https://firmware-selector.immortalwrt.org/?version=SNAPSHOT&target=mediatek%2Ffilogic&id=nokia_ea0326gmp)下载固件；
3. 刷新uboot
```
mtd write xxx-preloader.bin BL2
mtd write xxx-bl31-uboot.fip FIP
```
这步中preloader无法写入，忽略了，据说没关系；
4. 利用tftpd64恢复 initramfs-kernel固件；
5. ssh进入系统后sysgrade；

目前卡在卡在第五步;

```
root@ImmortalWrt:/tmp# sysupgrade -F -n immortalwrt-mediatek-filogic-nokia_ea0326gmp-squashfs-sysupgrade.itb
Thu Jan  1 00:02:56 UTC 1970 upgrade: Commencing upgrade. Closing all shell sessions.
Command failed: Connection failed
root@ImmortalWrt:/tmp# Connection to 192.168.1.1 closed by remote host.
Connection to 192.168.1.1 closed.
```
执行结果路由器重启，还是进入恢复模式；

有什么建议吗？
感激~

**Comment by xlighting2017** [comment]
接串口，看日志，否则不知道

**Reply by Tian-Jionglu** [reply]
接串口，看来得买新设备了~

**Comment by Tian-Jionglu** [comment, marked as answer]
问题以解决，见[715173329:master-mediatek-nokia](https://github.com/openwrt/openwrt/pull/14993)
